addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -70,6 +70,7 @@ def _needs_web_services(self):
     def _l10n_in_edi_xml_invoice_content(self, invoice):
         return json.dumps(self._l10n_in_edi_generate_invoice_json(invoice)).encode()
 
+    @api.model
```

Comment:
unnecessary change (nothing to do with ewaybill)

Reply:
But currently we are using this method in e waybill if `@api.model` doesn't makes much difference fine for me